### ISSUE TITLE
Error when importing Eslint issue with range of zero length

### ISSUE THREAD

**Issue by nicolas-harraudeau-sonarsource** [SonarSource/SonarJS#1026]
**I want to report a bug.**

**SonarJS version:** master branch. Build 4.2.0.6433 <br>
**SonarQube version:** 7.2<br>

**Origin:** Import of eslint reports.

**Reproducer**
The report is available [here](https://github.com/nicolas-harraudeau-sonarsource/eslint-report-integration-test/blob/8d5c2f2e71ff090619c990aa4ee73f8ce3f90a46/eslint_report.json). It can be recreated like this:
```sh
$ yarn install
$ yarn eslint
```
Then you can run the `sonnar-scanner`. In my case it fails with the output visible below (see Logs).

Here is a copy of the eslint report:
```
[
    {
        "filePath": "/Users/nicolasharraudeau/sonarsource/sonarjs-eslint-reports/jscode/src/index.js",
        "messages": [
            {
                "ruleId": "indent",
                "severity": 2,
                "message": "Expected indentation of 4 spaces but found 0.",
                "line": 2,
                "column": 1,
                "nodeType": "Identifier",
                "endLine": 2,
                "endColumn": 1,
                "fix": {
                    "range": [
                        15,
                        15
                    ],
                    "text": "    "
                }
            }
        ],
        "errorCount": 1,
        "warningCount": 0,
        "fixableErrorCount": 1,
        "fixableWarningCount": 0,
        "source": "function a() {\na()\n}"
    }
]
```

The sonnar-scanner succeeds if I set `endColumn` to 2.

**Logs**
sonar-scanner output
```
INFO: Sensor Import of ESLint issues [javascript]
INFO: Importing /Users/nicolasharraudeau/sonarsource/sonarjs-eslint-reports/jscode/./eslint_report.json
INFO: ------------------------------------------------------------------------
INFO: EXECUTION FAILURE
INFO: ------------------------------------------------------------------------
INFO: Total time: 9.663s
INFO: Final Memory: 37M/464M
INFO: ------------------------------------------------------------------------
ERROR: Error during SonarQube Scanner execution
ERROR: Start pointer [line=2, lineOffset=0] should be before end pointer [line=2, lineOffset=0]
ERROR:
ERROR: Re-run SonarQube Scanner using the -X switch to enable full debug logging.
```

**Expected behavior**

SUCCESS of the import
